CONTRIBUTING.md

```diff
@@ -31,6 +31,35 @@ Please use GitHub Discussions to submit feedback and engage with community [http
 
 ## Basic guidelines
 
+### Commits
+
+-  Use the imperative, present tense («change», not «changed» or «changes») to be consistent with generated messages from commands like git merge.
+- Describe the changes you have made
+
+#### Examples
+**Good example**: 
+ - Commit Message:\
+  `Add feature to calculate total monthly cost for AWS resources`
+
+- Description:\
+`Implement a function that calculates the total monthly cost for AWS resources. Update the documentation to reflect this new feature.`
```

Comment:
I'd make it into the same section (not separating the Commit Message & description), may be add an explanation that the first line is the title, and the description is the one that's placed after a newline after the title because in git it's like that

```
Add feature to calculate total monthly cost for AWS resources

Implement a function that calculates the total monthly cost for AWS resources.
Update the documentation to reflect this new feature.
```